tests/drivers/rest.sh

```diff
@@ -204,27 +211,48 @@ send_rest_go_command_expect_error_callback() {
   return 0
 }
 
+bypass_continues() {
+  status_line_idx=1
+  status_code=""
+  while true; do
+    status_line=$(sed -n "${status_line_idx}p" <<< "$result")
+    status_code=$(echo "$status_line" | awk '{print $2}')
+    if [ "$status_code" != "100" ]; then
+      break
+    fi
```

Comment:
This is a risky loop if this status is never reached.  Can you put in a limit of some sort so that you can ensure that it doesn't get stuck?